### PR TITLE
[Messenger] Ignore stamps in in-memory transport

### PR DESCRIPTION
**Pull request by tienvx**
| Q             | A
| ------------- | ---
| Branch?       | 4.4 
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Tickets       | N/A 
| License       | MIT
| Doc PR        | N/A 

Stamps can be added/removed to/from message during handling. Ignore stamps so that we can ack/reject them correctly.

### CODE REVIEW

File: src/Symfony/Component/Messenger/Tests/Transport/InMemoryTransportTest.php [hunk 1]
```diff
@@ -50,6 +51,19 @@ public function testQueue()
         $this->assertSame([], $this->transport->get());
     }
 
+    public function testQueueWithStamp()
```
Comment (sroze) [inline src/Symfony/Component/Messenger/Tests/Transport/InMemoryTransportTest.php]:
```suggestion
    public function testAcknowledgeSameMessageWithDifferentStamps()
```

**Review by sroze — COMMENTED** [review]
Looks legit indeed. Can you rename the test however so it describes what it's testing?

File: src/Symfony/Component/Messenger/Tests/Transport/InMemoryTransportTest.php (hunk 1 above)
Comment (tienvx) [reply src/Symfony/Component/Messenger/Tests/Transport/InMemoryTransportTest.php]: Thanks! Updated.

**Review by xabbuh — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Review by sroze — APPROVED** [review]

**Comment by sroze** [comment]
Thank you @tienvx.

**Pull request merged**